TraitsExpanded/GameStatusInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿using TaleWorlds.Core;
+
+namespace TraitsExpanded
+{
+    public class GameStatusInfo
```

Comment:
what's the purpose of this class?

Reply:
This is the data that passes through the GameTick, that the TraitSystems can subscribe to

Reply:
can't they get access to this information without it being passed?

Reply:
Yes, but I figured that down the line, we may want to relay some other data related to our mod specifically. Maybe we decide that we want to keep track of something else and send that through